cdap-examples/LogAnalysis/src/main/java/co/cask/cdap/examples/loganalysis/ApacheAccessLog.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
This class is taken from here: 
http://databricks.gitbooks.io/databricks-spark-reference-applications/content/logs_analyzer/chapter1/spark.html

http://databricks.gitbooks.io/databricks-spark-reference-applications/content/logs_analyzer/chapter1/java8/src/main/java/com/databricks/apps/logs/ApacheAccessLog.java

There is no attached license to it as such. @awholegunch : Should we say something here ?